jest.config.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+    // Configure jest-junit reporter on CI. Otherwise use only default reporter
+    reporters: [
+        'default',
+        ...(Boolean(process.env.CI)
+            ? [
+                  [
+                      'jest-junit',
```

Comment:
This may be redundant - you should what type of reports is supported by Travis

Reply:
Removed, will work on it after integrating with travis.